Unable to run web image

See below:
```
$ kubectl logs -n reach web-6ff6f6d7fd-gwzft
[2019-11-05 14:07:33 +0000] [1] [INFO] Starting gunicorn 19.9.0
[2019-11-05 14:07:33 +0000] [1] [INFO] Listening at: http://0.0.0.0:8080 (1)
[2019-11-05 14:07:33 +0000] [1] [INFO] Using worker: sync
[2019-11-05 14:07:33 +0000] [10] [INFO] Booting worker with pid: 10
/usr/local/lib/python3.6/site-packages/requests/__init__.py:91: RequestsDependencyWarning: urllib3 (1.25.2) or chardet (3.0.4) doesn't match a supported version!
  RequestsDependencyWarning)
[2019-11-05 14:07:35 +0000] [10] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.6/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/src/policytool/web/wsgi.py", line 2, in <module>
    from . import api
  File "/src/policytool/web/api.py", line 22, in <module>
    raise Exception("No static directory found. STATIC_ROOT=%r" % STATIC_ROOT)
Exception: No static directory found. STATIC_ROOT='/src/reach/web/build/static'
[2019-11-05 14:07:35 +0000] [10] [INFO] Worker exiting (pid: 10)
[2019-11-05 14:07:35 +0000] [1] [INFO] Shutting down: Master
[2019-11-05 14:07:35 +0000] [1] [INFO] Reason: Worker failed to boot.
```
Spotted on https://github.com/wellcometrust/datalabs-infra/pull/291#issuecomment-549838752

--- a/reach/web/Dockerfile.node
+++ b/reach/web/Dockerfile.node
@@ -17,6 +17,6 @@ RUN \
         postcss-import \
         postcss-url
 
-VOLUME ["/src/build/static", "/src/gulpfile.js", "/src/static"]
+VOLUME ["/build/web/static", "/src/gulpfile.js", "/src/static"]
 
 CMD ["gulp", "watch"]

--- a/reach/web/gulpfile.js
+++ b/reach/web/gulpfile.js
@@ -24,7 +24,7 @@ exports.css = function() {
     .pipe( postcss(plugins) )
     // sourcemaps are rooted in dest(), so '.' is what we want
     .pipe( sourcemaps.write('.') )
-    .pipe( dest('/src/build/static') );
+    .pipe( dest('/build/web/static') );
 };
 
 exports.default = parallel(exports.css);

